### PR TITLE
update genome nexus API client

### PR DESCRIPTION
**Pull request by leexgh**
update genome nexus API client

### CODE REVIEW

**Review by inodb — CHANGES_REQUESTED** [review]
@leexgh 	Thanks! Could you remove the `.DS_Store` file? Feel free to add a `.gitignore` file to this pull request, similar to: https://github.com/genome-nexus/genome-nexus/blob/master/.gitignore

**Comment by leexgh** [comment]
@inodb thanks! updated

**Review by inodb — APPROVED** [review]

File: genomeNexusInternalApiClient/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (onursumer) [inline genomeNexusInternalApiClient/.project]: we shouldn't probably include this one.

**Review by onursumer — CHANGES_REQUESTED** [review]

File: genomeNexusInternalApiClient/.project (hunk 1 above)
Comment (leexgh) [reply genomeNexusInternalApiClient/.project]: you mean the `.project` file?

Comment (inodb) [reply genomeNexusInternalApiClient/.project]: yes

File: genomeNexusPublicApiClient/.project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (leexgh) [inline genomeNexusPublicApiClient/.project]: @inodb @onursumer I also found this `.project` file, I'll remove both `.project` files and add `.gitignore` into PR

**Comment by leexgh** [comment]
@inodb @onursumer I have removed the `.project` files and updated the `.gitignore`

**Review by onursumer — APPROVED** [review]

**Pull request merged**